client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -161,6 +167,14 @@ class Upload extends React.Component {
 		const file = files[ 0 ] || files.item( 0 );
 		debug( 'zip file:', file );
 
+		if ( file.size && file.size > MAX_UPLOADED_THEME_SIZE ) {
```

Comment:
Do you need the property existence check here? `( undefined > MAX_UPLOADED_THEME_SIZE ) === false`